cmd/echo/echo.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+func main() {
+	var newLineFlag bool
+
+	flag.BoolVar(&newLineFlag, "n", false, "New Line")
+
+	flag.Parse()
+
+	args := flag.Args()
+
+	if len(args) != 0 {
+		for i := range len(args) {
+			fmt.Printf("%s ", args[i])
+		}
```

Comment:
if n flag is added a new line should be added between each argument
```suggestion
		}
		if newLineFlag {
				fmt.Printf("\n")
			}
```